README.md

```diff
@@ -15,6 +15,31 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
+## Development tools
+
+The `:dev` leiningen profile includes a dependency to the library `witan.workspace-executor`.
+This library contains tool that are useful for developer contributing to `witan.models.demography`.
```

Comment:
This isn't necessarily clear why this is important to someone who know nothing about this project. Also minor typo, should be tools not tool :)